processor/routingprocessor/processor_test.go

```diff
@@ -795,14 +946,16 @@ func (m *mockLogsExporter) getLogCount() int {
 type mockTracesExporter struct {
```

Comment:
I think we have a test facility native to the collector that we could use, right? Doesn't have to be changed on this PR, but perhaps it would make sense to open an issue to track this?

Reply:
Sure. We can always create an issue out of a comment. Do you happen to remember where does that live to reference it in the issue?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/blob/620c88ce0fc34cfbf0e0a0c444641ca75720ea6e/consumer/consumertest/sink.go#L25-L45

Reply:
Created an issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9125